virttest/test_setup.py

```diff
@@ -799,7 +799,13 @@ def __init__(self, params=None):
             if u_port not in ports:
                 ports.append(u_port)
             self.iptables_rules = self._assemble_iptables_rules(ports)
-            self.physical_nic = params.get("physical_nic")
+            if process.system(params.get("physical_nic"),
+                              shell=True, ignore_status=True) == 0:
+                self.physical_nic = decode_to_text(
+                        process.system_output(params.get("physical_nic"),
+                                              shell=True))
+            else:
+                self.physical_nic = params.get("physical_nic")
```

Comment:
You might exectue `params.get("physical_nic")` only once and also execute process.system_output() only once.

Reply:
yes, it looks repetitive．I have update the code. please help a review. thanks